src/cleanup_rules/ruby/rules.toml

```diff
@@ -0,0 +1,528 @@
+
+# Before : 
+# fields_array << :field_name unless false
+# After :
+# fields_array << :field_name
+[[rules]]
+name = "simplify_unless_statement_inline_false"
+groups = ["if_cleanup"]
+is_seed_rule = false
+query = """
+(
+  (unless_modifier
+    body : ([(binary) (identifier)] @body)
+    condition: (false)
+  )@unless_modifier
+)
+"""
+replace = "@body"
+replace_node = "unless_modifier"
+
+# Before : 
+# unless true
+#   do_something
+# else
+#   do_something_else
+# end
+# After :
+#  do_something_else
+[[rules]]
+name = "simplify_unless_statement_true"
+groups = ["if_cleanup"]
+is_seed_rule = false
+query = """
+(unless
+    condition: (true)
+)@unless
+"""
+replace = ""
+replace_node = "unless"
+
+
+# Before : 
+# unless false
+#   do_something
+# else
+#   do_something_else
+# end
+# After :
+#  do_something
+[[rules]]
+name = "simplify_unless_statement_false"
+groups = ["if_cleanup"]
+is_seed_rule = false
+query = """
+(unless
+    condition: (false)
+    consequence: ((then) @consequence)
+)@unless
+
+"""
+replace = "@consequence"
+replace_node = "unless"
+
+# Before :
+#  true ? flag_flow : non_flag_flow
+# After :
+#  flag_flow
+#
+[[rules]]
+name = "simplify_ternary_operator_true"
+groups = ["if_cleanup"]
+is_seed_rule = false
+query = """
+(
+  (conditional
+      condition : (true)
+      consequence : (_)* @consequence
+      alternative: (_)* @alternative
+      )
+@conditional)
+"""
+replace = "@consequence"
+replace_node = "conditional"
+
+# Before :
+#  false ? flag_flow : non_flag_flow
+# After :
+#  non_flag_flow
+#
+[[rules]]
+name = "simplify_ternary_operator_false"
+groups = ["if_cleanup"]
+is_seed_rule = false
+query = """
+(
+  (conditional
+      condition : (false)
+      consequence : (_)* @consequence
+      alternative: (_)* @alternative
+      )
+@conditional)
+"""
+replace = "@alternative"
+replace_node = "conditional"
+
+# Before : 
+#  !false
+# After :
+#  true 
+#
+[[rules]]
+groups = ["boolean_expression_simplify"]
+is_seed_rule = false
+name = "simplify_not_false"
+query = """
+(
+    (unary
+        operand: (false))
+@unary_expression)
+"""
+replace = "true"
+replace_node = "unary_expression"
+
+# Before : 
+#  !true
+# After :
+#  false
+#
+[[rules]]
+groups = ["boolean_expression_simplify"]
+is_seed_rule = false
+name = "simplify_not_true"
+query = """
+(
+    (unary
+        operand: (true))
+@unary_expression)
+"""
+replace = "false"
+replace_node = "unary_expression"
+
+# Before: 
+# (true)
+# After 
+# true
+
+[[rules]]
+name = "simplify_parenthesized_expression"
+query = "(parenthesized_statements ([(true) (false) (identifier)] @expression)) @p_expr"
+replace = "@expression"
+replace_node = "p_expr"
+is_seed_rule = false
+groups = ["boolean_expression_simplify"]
+
+# Before :
+#  abc() && true
+# After :
+#  abc()
+#
+[[rules]]
+groups = ["boolean_expression_simplify"]
+is_seed_rule = false
+name = "simplify_something_and_true"
+query = """
+(
+  (binary
+    left : (_) @lhs
+    operator: "&&"
```

Comment:
I think `ruby` supports `and` too right?

Reply:
done